catalyst-gateway/tests/schema-mismatch/Earthfile

```diff
@@ -0,0 +1,34 @@
+VERSION --global-cache 0.7
+
+builder:
+    FROM github.com/input-output-hk/catalyst-ci/earthly/python:v2.7.0+python-base
+
+    COPY --dir ./schema_mismatch README.md .
+    DO github.com/input-output-hk/catalyst-ci/earthly/python:v2.7.0+BUILDER
+
+package-tester:
+    FROM +builder
+
+    CMD poetry run pytest
+    # The following is useful for debugging the tests
+    # CMD poetry run pytest -vvvv --capture tee-sys --show-capture=stderr
+
+test:
+    FROM earthly/dind:alpine
```

Comment:
Actually you can modify this as follows

```Earthfile
FROM +builder
....
WITH DOCKER
        ...
        
        RUN poetry run pytest

END
```
Without dockerising `package-tester` tester, just provide all needed environment for the `test` target itself

Reply:
I tried that, but the most reliable and repeatable way to run is with the docker-compose services, specifically due to the `healthcheck` and `depends_on` values, which let the tests run once the services are healthy.